win/rl/pynethack.cc

```diff
@@ -122,6 +122,10 @@ class Nethack
         }
         strncpy(settings_.scoreprefix, scoreprefix.c_str(),
                 scoreprefix.length());
+        std::size_t found = ttyrec.rfind("/");
+        if (found!=std::string::npos && found + 1 < ttyrec.length())
```

Comment:
why the `found+1` check? `found` points to the "/", so in case someone names their ttyrec "/path/ttyrec.bz2/". Highly unlikely - maybe it would break? Will check.